generate/schema/index.js

```diff
@@ -43,18 +48,54 @@ export default function generateSchema(inputSchema) {
       inputTypeModifier = '!';
     }
 
+    // Simple Fields
     if (possibleInputType.kind === 'NamedType') {
       const isScalarField = includes(SCALAR_TYPE_NAMES, possibleInputType.name.value);
-      let inputField;
+      let CreateInputField = '';
+      let UpdateInputField = '';
       if (isScalarField || !!directivesByName.enum) {
-        inputField = field;
+        // take field as it was entered
+        CreateInputField = field;
+        // on updates, on NonNullable entered fields, the user should be able to decide, if he wants to update
```

Comment:
Best not to use gender specific language--make it eg. "the user should be able to decide, if they want to update"